### DISCUSSION TITLE
Bind to RH807 drone

### DISCUSSION THREAD

**Discussion by FabianInostroza** [pascallanger/DIY-Multiprotocol-TX-Module#706] [Q&A]
Hi,

I've got this small cheap drone
https://www.alibaba.com/product-detail/2020-RH807-RC-Drone-Mini-Quadcopter_1600155798053.html

The controller uses the RF2518 chip and the receiver uses the RF25155 chip, here are some photos:
![DSC_0658](https://user-images.githubusercontent.com/8893928/179655283-31523a2e-27b4-4d86-942b-9728003ef7dc.jpg)
![DSC_0660](https://user-images.githubusercontent.com/8893928/179655346-6a735fd9-bdf1-4446-9387-539aa09313c1.jpg)
![DSC_0663](https://user-images.githubusercontent.com/8893928/179655374-56b751c1-c8cb-4094-b51e-4ff0451e6070.jpg)

I have tried binding using the E012, E015, E129 and E130 protocols with no luck.
Any other protocol suggestions?
Should the XN297 dumper work for these chips? I have no SDR but I own a Jumper T Pro with internal JP4IN1 module that I can connect to the computer, so maybe I can dump some packets and give it a try.

**Comment by pascallanger** [comment]
Sorry but without knowing the addresses being used there is nothing that can be done (at least not remotely).
I need access to the TX and quad to reverse the protocol. Either send it to my attention in France or make a donation so I can purchase 1 sample.

**Reply by FabianInostroza** [reply]
Hi, I've made a donation through paypal.
This article claims to be the same I've got: https://www.aliexpress.com/item/1005004309438411.html?spm=a2g0o.cart.0.0.58cb7a9dlxfaFZ&mp=1&gatewayAdapt=glo2esp

I purchased a JJRC H36 but it was out of stock and the seller offered me the RH807 and I agreed. Now that seller has increased the price.

Let me know if I can help.

**Reply by pascallanger** [reply]
Hi,
I've placed the order. Let's wait for the delivery now.
Can you open an issue to request this new protocol to be added instead of a discussion? It's easier for me to manage.
- Pascal

**Reply by FabianInostroza** [reply]
Ok, thanks!

**Comment by bla671** [comment]
any news on how to use this drone with multiprotocol radio?

**Reply by FabianInostroza** [reply]
I replaced the fly controller

**Reply by bla671** [reply]
what fly controller did you replace the stock one with? can i get a link to it? :)

**Reply by FabianInostroza** [reply]
https://es.aliexpress.com/item/32847226537.html

**Reply by ClawMarksss** [reply]
hi, the link is outdated, could you please send me the title?
